Userland/Libraries/LibCpp/AST.h

```diff
@@ -60,6 +61,10 @@ class ASTNode : public RefCounted<ASTNode> {
         ASSERT(m_end.has_value());
         return m_end.value();
     }
+    StringView filename() const
+    {
+        return m_filename.view();
+    }
```

Comment:
If we're already storing a `String` we might as well return a `const String&` here, no point in only exposing a `StringView`.

Reply:
Well we store`m_filename` as a `FlyString` (to avoid having copies of the same filename in all of the `ASTNode`s). Is there an alternative to exposing a `StringView` here?

Reply:
Sure, return a `const FlyString&` :)